.github/workflows/pr.yml

```
@@ -56,22 +76,57 @@
+          # Build versioned docs with sphinx-versioned-docs
+          sphinx-versioned \
+            --output-dir ./versioned-docs \
+            --git-root . \
+            --source-dir projects \
+            --branches main,${{ env.BRANCH_SLUG }} \
+            --show-banner \
+            --banner-main-ref main \
+            --banner-newest-tag \
+            --invert \
+            --verbose
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)